.github/workflows/node.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  builds:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    
+    - name: Install dependencies
+      working-directory: ./ui
+      run: yarn install
+    - name: Build 
+      working-directory: ./ui
+      run: yarn run build-prod
+    - name: Linting
+      working-directory: ./ui
+      run: |
+        yarn lint:prettier
```

Comment:
Let's split this up into two jobs, `Lint Prettier` and `Lint Eslint` so it's easier for us to see what failed